script/QA/QA_16.txt

```diff
@@ -228,6 +228,7 @@ We gotta make sure we choose Arcueid!
 -- Page 95, Offset 43494.
 -- ――数時間後――
 // lint-off:LintEmDashes
+// My game softlocks here and doesn't show the last two lines
```

Comment:
Should also be fixed if you update allscr, or do you still see it after doing that?